.travis.yml

```diff
@@ -13,15 +14,19 @@ matrix:
 
 before_install:
   # on OSX, we use homebrew to install python 3.5.2
-  - if [ "$TRAVIS_OS_NAME" == "osx" ]; then brew update || brew update; fi
-  - if [ "$TRAVIS_OS_NAME" == "osx" ]; then if test $(brew list | grep -q 'python3'); then brew upgrade python3; else brew install python3; fi; fi
+  - if [ "$TRAVIS_OS_NAME" == "osx" ]; then wget https://www.python.org/ftp/python/3.5.4/python-3.5.4-macosx10.6.pkg; fi
+  - if [ "$TRAVIS_OS_NAME" == "osx" ]; then sudo installer -store -pkg python-3.5.4-macosx10.6.pkg -target /; fi
   # check environment
   - echo $PATH
   - which python3
+  - python3 --version
   - pip3 --version
   - pip3 list
   # install requirements
-  - pip3 install --upgrade pip setuptools wheel
+  - python3 -m venv venv
+  - . venv/bin/activate
+  - curl https://bootstrap.pypa.io/get-pip.py | python3
+  - pip3 install --upgrade setuptools wheel
```

Comment:
looks like you don't get `pip3` when running get-pip.py on linux: https://travis-ci.org/trufont/trufont/jobs/370693421#L502

just call it `pip` (as you're in the venv) or `python3 -m pip`